controllers/topics.controller.js

```diff
@@ -0,0 +1,13 @@
+const { selectAllTopics } = require("../models/topics.model");
+
+exports.getAllTopics = (req, res, next) => {
+  return selectAllTopics(req)
+    .then(({ rows }) => {
+      if (!rows) {
+        return Promise.reject({ status: 404, msg: "Topics not found" });
+      } else {
```

Comment:
We should be sending our rows across from our model, as well as doing any sort of Promise.reject in there as well. The controller should just be for invoking the model, then sending the response, if you could refactor please.